yarn-project/noir-contracts/scripts/compile.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Compiles Aztec.nr contracts in parallel, bubbling any compilation errors
+
+export self_dir=$(dirname "$(realpath $0)")
+export COMPILER="$self_dir/../../noir-compiler/dest/cli.js"
+
+build() {
+  CONTRACT_NAME=$1
+  CONTRACT_FOLDER="$self_dir/../src/contracts/${CONTRACT_NAME}_contract"
+  echo "Compiling $CONTRACT_NAME..."
+  rm -rf ${CONTRACT_FOLDER}/target
+
+  # If the compilation fails, rerun the compilation with 'nargo' and show the compiler output.
```

Comment:
Is this comment still relevant?

Reply:
oh, this file got deleted from master and I'm restoring it. I'll update the comment